matplotcheck/autograde.py

```diff
@@ -0,0 +1,89 @@
+# Wrapper functions for matplotcheck functions that run
+# and track the passing status of test functions and
+# provide formatted results of tests
+
+
+def run_test(
+    func,
+    points,
+    *args,
+    correct_message="default correct",
+    error_message="default error",
+    **kwargs
+):
+    """Runs a pre-defined test function and creates a dictionary
+    containing the results of the test
+
+    Parameters
+    ---------
+    func: name of pre-defined test function to run
```

Comment:
@jlpalomino please following standard docstring syntax like we do for numpy

look at the earthpy doc strings: https://github.com/earthlab/earthpy/blob/master/earthpy/clip.py#L20 
or just read about it. 

you should have the parameter type and the line below will have the description. this is important as we use autodoc here but also important for good documentation and standards in general.

Reply:
Corrected.